modules/policy-identity-access-management.adoc

```diff
@@ -25,7 +23,7 @@ Red Hat SRE adheres to the principle of least privilege when accessing {product-
 
 * SRE admin access through the Red Hat corporate SSO with normal two-factor authentication and no privileged elevation.
 
-* OpenShift elevation, which is a manual elevation using Red Hat SSO. It is limited to 2 hours, is fully audited, and requires management approval.
+* OpenShift elevation, which is a manual elevation using Red Hat SSO. It is fully audited, and requires management approval for every operations SRE made.
```

Comment:
You could remove the comma after "audited" and make "operations" singular. For example:

"It is fully audited and management approval is required for every operation SREs make."